tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
-  "extends": "@seek/seek-module-toolkit/config/tsconfig.json",
+  "extends": "@seek/skuba/config/tsconfig.json",
   "compilerOptions": {
     "declaration": true,
     "incremental": true,
     "module": "CommonJS",
+    "moduleResolution": "node",
```

Comment:
I double checked and seek-module-toolkit has an exact copy of the `tsconfig-seek` that `skuba` uses, except that this is explicitly set.